schema/Cargo.toml

```diff
@@ -7,8 +7,8 @@ edition = "2021"
 [dependencies]
 iroha_schema_derive = { version = "=2.0.0-pre.1", path = "derive" }
 
-serde = { version = "1.0", features = ["derive"] }
-fixnum = { version = "0.6", features = ["i64"]}
+serde  = { version = "1.0.132", default-features = false, features = ["derive", "alloc"] }
+fixnum = { version = "0.6.1",   default-features = false, features = ["i64"] }
 
 [dev-dependencies]
-parity-scale-codec = { version = "2", features = ["derive"] }
+parity-scale-codec = { version = "2.3.1", default-features = false, features = ["derive", "full"] }
```

Comment:
I wonder if we need `full` here. [This](https://github.com/paritytech/parity-scale-codec/blob/94f5887207af4abfb85e266985f9bd260ee09e04/Cargo.toml#L53) makes me think that it might be better to hide this behind a feature flag and exclude this feature from any code that interacts with `sumeragi`.

Reply:
In Parity Scale this is implied by `std`: `#[cfg(any(feature = "std", feature = "full"))]`

Reply:
I asked a question with parity, but I find it quite unlikely this should bother us

Reply:
Answer from parity:
> It is what I meant with this comment. When building blockchains and using Strings in the runtime shows that you are probably doing something wrong. Like you trying to store user data or whatever. In these cases it is better to hash the data and only store the hash.